tephra-hbase-compat-0.96/src/main/java/co/cask/tephra/hbase96/coprocessor/TransactionProcessor.java

```diff
@@ -149,10 +149,13 @@ public void preGetOp(ObserverContext<RegionCoprocessorEnvironment> e, Get get, L
     throws IOException {
     Transaction tx = getFromOperation(get);
     if (tx != null) {
+      boolean excludeCurrentWritePtr = isAttributeSet(get, TxConstants.TX_EXCLUDE_CURRENT_WRITE);
```

Comment:
@anew I don't follow your comment. Whether exclude or include current writepointer - is an attribute of an operation/request, not whole transaction. I think things are good here.